docs/user_guide.md

```diff
@@ -4,63 +4,146 @@ This is a pre-release version of the extension and is not intended for general u
+
+`pg_diffix.treat_unmarked_tables_as_public` - Determines the default security label for tables. If set to `true`, then the default security label is `public`.  The default value is `false`.
+
+### Anonymization strength settings
+
+Diffix has eight constants that determine the "strength" of anonymization. These impact the amount of noise, the number-of-persons threshold below which suppression takes place, and the behavior of flattening (see [this article](https://www.open-diffix.org/blog/diffix-elm-automates-what-statistics-offices-have-been-doing-for-decades) for an overview of these concepts).
+
+Seven of these eight constants are set to values that provide sufficiently strong anonymity in virtually any realistic scenario. These seven constants rarely if ever need to be modified. Doing so only reduces data quality with little meaningful strengthening of anonymity.
+
+The one anonymization constant that sometimes requires adjustment is `pg_diffix.low_count_min_threshold`. This sets the lower bound for the number of distinct AID values that must be present in a aggregate bucket to avoid suppression. Any buckets with fewer than `pg_diffix.low_count_min_threshold` distinct AIDs will be suppressed. See (How to set suppression threshold)[#how-to-set-suppression-threshold] for further guidance.
+
+Default value is 3. Minimum allowed setting is 2.
+
+#### Other seven settings
+
+`pg_diffix.noise_layer_sd` - Standard deviation for each noise layer added to aggregates. Default value is 1.0. Minimum allowed setting is 1.0.
+
+`pg_diffix.low_count_mean_gap` - The number of standard deviations between the lower bound `pg_diffix.low_count_min_threshold` and the mean of the
+low count filter threshold. Default value is 2.0. Minimum allowed setting is 2.0.
 
-At access levels other than `direct`, various data and features built into PostgreSQL are restricted. Among others:
+`pg_diffix.low_count_layer_sd` - The standard deviation for each noise layer used when calculating the low count filter threshold. Default value is 1.0. Minimum allowed setting is 1.0.
+
+`pg_diffix.outlier_count_min` - Default value is 1. Must not be greater than `outlier_count_max`. Minimum allowed setting is 1.
+
+`pg_diffix.outlier_count_max` - Default value is 2. Must not be smaller than `outlier_count_min`. Minimum allowed setting is 2.
+
+`pg_diffix.top_count_min` - Default value is 3. Must not be greater than `top_count_max`. Minimum allowed setting is 2.
+
+`pg_diffix.top_count_max` - Default value is 4. Must not be smaller than `top_count_min`. Minimum allowed setting is 3.
+
+**NOTE** The outlier interval `(outlier_count_min, outlier_count_max)` must not be wider than the `(top_count_min, top_count_max)` interval.
```

Comment:
@pdobacz has made changes here and this line is redundant afaik.

Reply:
ok